<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="lb_Author">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Author:Light&amp;Electricity</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTime_getCreat">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd HH:mm:ss</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fileName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>530</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTime_getMod">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd HH:mm:ss</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTime_getAccess">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd HH:mm:ss</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTime_setMod">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="date">
    <date>
     <year>2020</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd HH:mm:ss</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTime_setCreat">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="date">
    <date>
     <year>2020</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd HH:mm:ss</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTime_setAccess">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="date">
    <date>
     <year>2020</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd HH:mm:ss</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>创建时间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_open">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>打 开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_set">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>60</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>修 改</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>当前</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>访问时间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_set_Creat">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>120</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_copy_Creat">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>→</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_copy_Mod">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>→</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_copy_Access">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>→</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_set_Mod">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>160</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_set_Access">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>200</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_set_instruction">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>说 明</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>580</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="sBox_rangeC">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="suffix">
    <string>s</string>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="minimum">
    <number>-86400000</number>
   </property>
   <property name="maximum">
    <number>86400000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sBox_rangeM">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="suffix">
    <string>s</string>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="minimum">
    <number>-86400000</number>
   </property>
   <property name="maximum">
    <number>86400000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sBox_rangeA">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="suffix">
    <string>s</string>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="minimum">
    <number>-86400000</number>
   </property>
   <property name="maximum">
    <number>86400000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>255</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>增减量/波动范围</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>创建时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>访问时间</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_ifC">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_ifM">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_ifA">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_set_multiFileInc">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>时间递进</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_set_multiFileRand">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>随机波动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_set_instructionM">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>360</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>说 明</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>多文件操作</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
